MANUAL.md

```diff
@@ -0,0 +1,194 @@
+# SHIELD Genesis Kit Manual
+
+The **SHIELD Genesis Kit** deploys the SHIELD Cloud Data
+Protection suite, to allow you to back up your sensitive cloud
+infrastructure and application data.
+
+# Base Parameters
+
+- `shield_static_ip` - The static IP address of the SHIELD core.
+  This must be designated as static in the chosen network's cloud
+  config definition.
+
+- `installation` - The name of the SHIELD installation, as
+  reported from inside the SHIELD UI.
+  Defaults to `S.H.I.E.L.D. Alpha`
```

Comment:
Actually this defaults to `S.H.I.E.L.D.`

Reply:
Actually, it technically doesn't default to anything (ie it is not defined), but if not present, things that look it up default to it just being `S.H.I.E.L.D.`

Reply:
:+1: